### PR TITLE
Update hosts.txt

### PR DESCRIPTION
**Pull request by FlyingBackdoor**
Added some new and proxy domains

### CODE REVIEW

File: submit_here/hosts.txt [hunk 1]
```diff
@@ -12947,3 +12948,35 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+#Submission
```
Comment (spirillen) [inline submit_here/hosts.txt:12951]: Invalid Line...

It is absolutely a long list (Bulk) but without individual proof, this PR will be awaiting for @Clefspeare13 

Best alternative is you submit each one as issues here: <https://www.mypdns.org/maniphest/task/edit/form/2/>

With screenshots....

**Review by spirillen — CHANGES_REQUESTED** [review]

**Review by FlyingBackdoor — COMMENTED** [review]
Removed Invalid line

File: submit_here/hosts.txt [hunk 2]
```diff
@@ -10638,6 +10638,7 @@ www.porndish.com
 www.porndisney.com
 www.porndoe.com
 www.porndoepremium.com
+www.porndude.me
```
Comment (spirillen) [inline submit_here/hosts.txt:10641]: https://www.mypdns.org/T1923

File: submit_here/hosts.txt [hunk 3]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
+xnxx-films.com
+xnxxvideos.fun
+xnxxporn.video
+xnxxporn.top
+xnxx123.org
+xnxxmommy.com
+indian-xnxx.pro
```
Comment (spirillen) [inline submit_here/hosts.txt:12981]: https://www.mypdns.org/T4429

File: submit_here/hosts.txt [hunk 4]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
+xnxx-films.com
+xnxxvideos.fun
+xnxxporn.video
+xnxxporn.top
+xnxx123.org
+xnxxmommy.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12980]: https://www.mypdns.org/T4430

File: submit_here/hosts.txt [hunk 5]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
+xnxx-films.com
+xnxxvideos.fun
+xnxxporn.video
+xnxxporn.top
+xnxx123.org
```
Comment (spirillen) [inline submit_here/hosts.txt:12979]: https://www.mypdns.org/T4431

File: submit_here/hosts.txt [hunk 6]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xnxx.news
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
+xnxx-films.com
+xnxxvideos.fun
+xnxxporn.video
+xnxxporn.top
```
Comment (spirillen) [inline submit_here/hosts.txt:12978]: https://www.mypdns.org/T4432

File: submit_here/hosts.txt [hunk 7]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xnnx.name
+xnxx.news
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
+xnxx-films.com
+xnxxvideos.fun
+xnxxporn.video
```
Comment (spirillen) [inline submit_here/hosts.txt:12977]: https://www.mypdns.org/T4433

File: submit_here/hosts.txt [hunk 8]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xnxx.college
+xnnx.name
+xnxx.news
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
+xnxx-films.com
+xnxxvideos.fun
```
Comment (spirillen) [inline submit_here/hosts.txt:12976]: https://www.mypdns.org/T4434

File: submit_here/hosts.txt [hunk 9]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xnxxhd.biz
+xnxx.college
+xnnx.name
+xnxx.news
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
+xnxx-films.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12975]: https://www.mypdns.org/T4435

File: submit_here/hosts.txt [hunk 10]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
+xnnx.name
+xnxx.news
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
+xnxxx.rocks
```
Comment (spirillen) [inline submit_here/hosts.txt:12974]: https://www.mypdns.org/T4436

File: submit_here/hosts.txt [hunk 11]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
+xnnx.name
+xnxx.news
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
+www.xnxx.report
```
Comment (spirillen) [inline submit_here/hosts.txt:12973]: https://www.mypdns.org/T4437

File: submit_here/hosts.txt [hunk 12]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
+xnnx.name
+xnxx.news
+xnxx.sexy
+xnxx-best.com
+xnxxbest.pro
```
Comment (spirillen) [inline submit_here/hosts.txt:12972]: https://www.mypdns.org/T4438

File: submit_here/hosts.txt [hunk 13]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
+xnnx.name
+xnxx.news
+xnxx.sexy
+xnxx-best.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12971]: Needs the `www.$domain`

https://www.mypdns.org/T4439

File: submit_here/hosts.txt [hunk 14]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
+xnnx.name
+xnxx.news
+xnxx.sexy
```
Comment (spirillen) [inline submit_here/hosts.txt:12970]: https://www.mypdns.org/T4440

File: submit_here/hosts.txt [hunk 15]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
+xnnx.name
+xnxx.news
```
Comment (spirillen) [inline submit_here/hosts.txt:12969]: https://www.mypdns.org/T4445

File: submit_here/hosts.txt [hunk 16]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
+xnnx.name
```
Comment (spirillen) [inline submit_here/hosts.txt:12968]: https://www.mypdns.org/T4446

File: submit_here/hosts.txt [hunk 17]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
+xnxx.college
```
Comment (spirillen) [inline submit_here/hosts.txt:12967]: https://www.mypdns.org/T4447

File: submit_here/hosts.txt [hunk 18]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
+xnxxhd.biz
```
Comment (spirillen) [inline submit_here/hosts.txt:12966]: https://www.mypdns.org/T4448

File: submit_here/hosts.txt [hunk 19]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
+www.xxnx.bike
```
Comment (spirillen) [inline submit_here/hosts.txt:12965]: https://www.mypdns.org/T4450

File: submit_here/hosts.txt [hunk 20]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
```
Comment (spirillen) [inline submit_here/hosts.txt:12963]: https://www.mypdns.org/T4451

I have also added `xhamster{1..11}.desi`

File: submit_here/hosts.txt [hunk 21]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12962]: https://www.mypdns.org/T4461

File: submit_here/hosts.txt [hunk 22]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12961]: https://www.mypdns.org/T1445

File: submit_here/hosts.txt [hunk 23]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
```
Comment (spirillen) [inline submit_here/hosts.txt:12960]: https://www.mypdns.org/T4462

File: submit_here/hosts.txt [hunk 24]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
 zzztube.com
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12959]: https://www.mypdns.org/T4463

File: submit_here/hosts.txt [hunk 25]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
 zzitube.com
 zzztube.com
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12958]: https://www.mypdns.org/T4464

File: submit_here/hosts.txt [hunk 26]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
+xfantazy.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12957]: https://www.mypdns.org/T4465

File: submit_here/hosts.txt [hunk 27]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
+www.dinotube.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12956]: https://www.mypdns.org/T4466

File: submit_here/hosts.txt [hunk 28]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
+porndude.xyz
```
Comment (spirillen) [inline submit_here/hosts.txt:12955]: https://www.mypdns.org/T4468

File: submit_here/hosts.txt [hunk 29]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
+www.thepornlist.net
```
Comment (spirillen) [inline submit_here/hosts.txt:12954]: https://www.mypdns.org/T4462

File: submit_here/hosts.txt [hunk 30]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+mrporndude.com
+theporndude.vip
+teenxnxx.cc
```
Comment (spirillen) [inline submit_here/hosts.txt:12953]: https://www.mypdns.org/T4470

File: submit_here/hosts.txt [hunk 31]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+mrporndude.com
+theporndude.vip
```
Comment (spirillen) [inline submit_here/hosts.txt:12952]: https://www.mypdns.org/T4471

File: submit_here/hosts.txt [hunk 32]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
 zzgays.com
 zzitube.com
 zzztube.com
+mrporndude.com
```
Comment (spirillen) [inline submit_here/hosts.txt:12951]: https://www.mypdns.org/T4472

File: submit_here/hosts.txt [hunk 33]
```diff
@@ -12947,3 +12948,34 @@ zzbabes.com
...
+porndude.xyz
+www.dinotube.com
+xfantazy.com
+pornlistdude.com
+porndudecasting.com
+www.thepornlist.net
+www.pornmd.com
+www.porn-xnxx.com
+xhamster7.desi
+xlxx.space
```
Comment (spirillen) [inline submit_here/hosts.txt:12964]: https://www.mypdns.org/T4473

**Pull request closed without merging**